### PR TITLE
Set AICoE CI to have socrates content image for experiments on ODH

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

**Review by xtuchyna — APPROVED** [review]
/approve thxx! :rocket:

File: thoth/socrates/__init__.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""This file carries the version of the template project."""
+"""Socrates."""
 
-__version__ = "0.1.0"
+__name__ = "thoth-socrates"
```
Comment (xtuchyna) [inline thoth/socrates/__init__.py:20]: maybe the question is, if this should be under thoth station or something else?

Comment (pacospace) [reply thoth/socrates/__init__.py:20]: only `socrates`?

**Review by harshad16 — COMMENTED** [review]
/lgtm
:100:

File: thoth/socrates/__init__.py (hunk 1 above)
Comment (xtuchyna) [reply thoth/socrates/__init__.py:20]: yes

**Comment by pacospace** [comment]
all seems ready! we can merge it and create a first release!

**Pull request merged**